basedtyping/runtime_only.py

```diff
@@ -4,9 +4,10 @@
 This is the similar to the ``types`` module.
 """
 
-from typing import Final, Literal, Union
+from typing import Union, Type
+from typing_extensions import Final, Literal
 
-LiteralType: Final = type(Literal[1])
+LiteralType: Final[Type[Literal[1]]] = type(Literal[1])  # type: ignore[no-any-expr]
```

Comment:
doesn't the type get inferred anyway?

Reply:
Woops, I was just testing that out. Didn't mean to commit.